content/contributing/opencue/Getting started/getting-started-linux.md

```diff
@@ -0,0 +1,312 @@
+   ```
+
+1.  Change your current directory to the root of your OpenCue Git clone.
+
+1.  Visit https://github.com/AcademySoftwareFoundation/OpenCue/releases and download the SQL files
+    from the latest release's Assets. There should be two - a `schema` file and a `demo_data` file.
+
+1.  Populate the database schema and some initial data, run the `psql`
+    command:
+
+    ```shell
+    psql -h $DB_HOST -f <path to schema SQL file> $DB_NAME
+    ```
+
+1.  The `demo_data` SQL file contains a series of SQL commands to insert some
+    sample data into the Cuebot database, which helps demonstrate various
+    features of the software. To execute the SQL statements, run the `psql`
+    command:
+
+    ```shell
+    psql -h $DB_HOST -f <path to demo_data SQL file> $DB_NAME
+    ```
+    To see a list of flags for the `psql` tool, run the `psql --help` command. For example, if
+    your database is running on a remote host, specify the `-h` flag. If you need to specify a
+    different username, specify the `-U` flag.
+
+## Running Cuebot
+
+Cuebot is the core component of OpenCue, written in Java. It serves the gRPC API which all
+other components use to interact with the database, and is responsible for dispatching work
+to RQD worker nodes.
+
+To build and run it with IntelliJ IDEA:
+
+1. Open IntelliJ IDEA and choose **Import Project**, select the `cuebot` folder in the git
+   repository.
+   
+1. Choose the **Import project from external model > Gradle** option then click **Finish**.
+
+1. Click **Build > Rebuild Project**.
+
+   IntelliJ downloads Gradle and all source dependencies, then compiles the project and
+   runs tests.
+
+1. Browse to the `src/main/java/com.imageworks/spcue/CuebotApplication` file.
+
+1. Click **Edit 'CubotApplicat....main()'...**.
+
+1. Update the **Program arguments** as follows and replace the value for `<PASSWORD>`
+   where indicated:
+
+   ```
+   --datasource.cue-data-source.jdbc-url=jdbc:postgresql://localhost/cuebot_dev --datasource.cue-data-source.username=cuebot --datasource.cue-data-source.password=<PASSWORD>
+   ```
+ 
+1. Click **OK**.
+ 
+1. Click **Run** > **Run 'CuebotApplication'**.
+
+1. Verify that the output window doesn’t show any errors.
+
+   If it does, double-check that you have set up the database correctly,
+   including permissions, and have set the Program arguments correctly.
+
+## Create a virtual environment
+
+OpenCue consists of multiple Python components which are interrelated. These components
+include RQD, CueGUI, and the Python API.
+
+We recommend creating a Python virtual environment specifically for development use that you can
+use for all components. This will help keep dependencies synchronized across your OpenCue
+deployment.
+
+1. Open a terminal and change to the root folder of your OpenCue Git clone.
+
+1. Create a virtual environment named `venv-dev`:
+
+   ```shell
+   sudo python3 -m venv venv-dev
```

Comment:
`sudo` shouldn't be necessary to create a virtual environment, I think.

Reply:
Sure, I will remove it.